employee-rostering-benchmark/src/main/resources/employeeRosteringBenchmarkConfig.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plannerBenchmark>
+  <benchmarkDirectory>local/benchmarkReport</benchmarkDirectory>
+  <warmUpSecondsSpentLimit>30</warmUpSecondsSpentLimit>
+
+  <inheritedSolverBenchmark>
+    <solver>
+      <solutionClass>org.optaweb.employeerostering.domain.roster.Roster</solutionClass>
+      <entityClass>org.optaweb.employeerostering.domain.shift.Shift</entityClass>
+
+      <scoreDirectorFactory>
+        <scoreDrl>org/optaweb/employeerostering/service/solver/employeeRosteringScoreRules.drl</scoreDrl>
+      </scoreDirectorFactory>
+
+      <termination>
+        <secondsSpentLimit>10</secondsSpentLimit>
+      </termination>
+    </solver>
+  </inheritedSolverBenchmark>
+
+  <solverBenchmark>
+    <name>Default</name>
+    <solver>
+    </solver>
+  </solverBenchmark>
```

Comment:
Personally, I'd add a comment saying that the block is intentionally empty. Otherwise people will wonder.